### PR TITLE
Add time the song is added to the data

### PR DESCRIPTION
**Pull request by muyjohno**
Also sort items when adding

### CODE REVIEW

File: src/marvin.js [hunk 1]
```diff
@@ -76,15 +78,18 @@ function Marvin(options) {
...
         this.queue.push(data, function(err, reply) {
           if(err) { return }
           console.log('added track', data)
 
-          if(!this.now_playing && this.master_player_id) {
-            this.get_next_track()
-          } else {
-            this.update_all_clients()
-          }
+          this.queue.sort(function() {
```
Comment (martinlockett) [inline src/marvin.js]: How does it know what to sort by?

Comment (muyjohno) [inline src/marvin.js]: The `marvin-queue` module contains the sorting logic (currently just by score)

Comment (martinlockett) [inline src/marvin.js]: Ah ok, I was thinking the idea was to sort by date too

**Comment by martinlockett** [comment]
👍

**Pull request merged**